pipelines/agg_matched_multi.yaml

```diff
@@ -0,0 +1,8 @@
+description: Compute aggregate metrics from a list of individual measurements
+tasks:
+  agg_mean_AB1_design:
```

Comment:
Hmm.  I don't see `AB1` in the `verify_metrics` package.  Do wee need to add it?

Reply:
Wow, I did that months ago, then forgot to merge it. PR coming your way...

Reply:
DM-22310 on verify_metrics

Reply:
Same as above with the task name.